src/pages/index.js

```diff
@@ -1,22 +1,31 @@
 import { makeStyles } from "@material-ui/core/styles"
 import { Typography } from "@material-ui/core"
 import { useSelector } from "react-redux"
+import { useState } from "react"
 
 import CustomTable from "@/components/CustomTable"
+import AddNewButton from "@/components/AddNewButton"
+import RightSideDrawer from "@/components/RightSideDrawer"
 import { characterSelectors } from "@/selectors/characterSelectors"
```

Comment:
Your selector should be named after the piece of state you are selecting. This is what your selector looks like
```
export const characterSelectors = (state) => state.character.characters
```

It should be 
```
export const characters = (state) => state.character.characters

```